src/legacy/ui/public/agg_types/buckets/filters.ts

```diff
@@ -67,8 +67,13 @@ export const filtersBucketAgg = new BucketAggType({
               return;
             }
 
-            const query = esQuery.buildEsQuery(aggConfig.getIndexPattern(), [input], [], config);
```

Comment:
The problem was that `config` which was passed here is `chrome.getUiSettingsClient()`, instead of `{ allowLeadingWildcards: config.get('query:allowLeadingWildcards') }`

So `allowLeadingWildcards` would default to `false`, and then fail when trying to parse the query.